packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -400,12 +403,28 @@ class TapDragEndDetails with Diagnosticable {
   /// the number in the series this tap is.
   final int consecutiveTapCount;
 
+  /// The pointer's global position when it triggered this update.
+  ///
+  /// See also:
+  ///
+  ///  * [localPosition], which is the [globalPosition] transformed to the
+  ///    coordinate space of the event receiver.
+  final Offset globalPosition;
+
+  /// The local position in the coordinate system of the event receiver at
```

Comment:
nit: maybe this should say `The local position in the coordinate system of the event receiver at which the pointer lifted from the screen.`